### PR TITLE
add issue templates

### PR DESCRIPTION
**Pull request by hsl89**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/example-request.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+labels: ''
+assignees: ''
+
+---
+
+**Describe the use case example you want to see**
+
+
+
+**Describe what SageMaker services are involved**
```
Comment (aaronmarkham) [inline .github/ISSUE_TEMPLATE/example-request.md:14]:
```suggestion
**Describe which SageMaker services are involved**
```

File: .github/ISSUE_TEMPLATE/example-request.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+
+
+
+**Describe what AWS services (other than SageMaker) are involved***
```
Comment (aaronmarkham) [inline .github/ISSUE_TEMPLATE/example-request.md:19]:
```suggestion
**Describe what other services (other than SageMaker) are involved***
```

File: .github/ISSUE_TEMPLATE/example-request.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+name: Example request
+about: Suggest an use case example you want to see
+title: I want to see ...
```
Comment (aaronmarkham) [inline .github/ISSUE_TEMPLATE/example-request.md:4]: Should this use a similar pattern to [Content Improvement]?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
```
Comment (aaronmarkham) [inline .github/ISSUE_TEMPLATE/bug_report.md:4]: [Bug report]?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+labels: ''
+assignees: ''
+
+---
+
+**Link to the notebook**
+Which notebook you run into trouble
```
Comment (aaronmarkham) [inline .github/ISSUE_TEMPLATE/bug_report.md:11]:
```suggestion
Add the link to the notebook
```

File: .github/ISSUE_TEMPLATE/example-request.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+labels: ''
```
Comment (aaronmarkham) [inline .github/ISSUE_TEMPLATE/example-request.md:5]: let's put a default label on these...

**Review by aaronmarkham — COMMENTED** [review]
Cool! Some suggestions for consistency.

File: .github/ISSUE_TEMPLATE/example-request.md (hunk 3 above)
Comment (hsl89) [reply .github/ISSUE_TEMPLATE/example-request.md:4]: That's right. Adding it now

**Review by aaronmarkham — APPROVED** [review]

**Pull request merged**